Sources/Datadog/URLSessionAutoInstrumentation/DDURLSessionDelegate.swift

```diff
@@ -0,0 +1,42 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2019-2020 Datadog, Inc.
+ */
+
+import Foundation
+
+@objc
```

Comment:
`DatadogObjc.framework` is an umbrella on top of `Datadog.framework`. Objective-C users only link `DatadogObjc` to their projects and all the symbols must be available in there. Without `DDURLSessionDelegate+objc.swift` they would have to link also Swift SDK to access only the delegate.

Reply:
okay i see 👍 
maybe if we add this file to `Datadog` and `DatadogObjC` targets at the same time it may work
but i guess then `podspec` files will become more complicated

resolved ✅